bill_test.go

```diff
@@ -0,0 +1,85 @@
+package elation
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBillService_Create(t *testing.T) {
+	testCases := map[string]struct {
+		create *BillCreate
+	}{
+		"required fields only request": {
+			create: &BillCreate{
+				ServiceLocation: &BillServiceLocation{},
+				VisitNoteID:     64409108504,
+				Patient:         64901939201,
+				Practice:        65540,
+				Physician:       64811630594,
+			},
+		},
+		"all specified fields request": {
```

Comment:
It might be nice for this test case to have actual values in the empty objects? but not blocking